<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>750</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(30, 30, 30);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>900</width>
      <height>750</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="importPg">
     <widget class="QLabel" name="importPg_lbl">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>640</y>
        <width>261</width>
        <height>81</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(86, 86, 86);</string>
      </property>
      <property name="text">
       <string>Step 1: Drag and Drop File</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="Drag" name="importPg_wid_fileDrg" native="true">
      <property name="geometry">
       <rect>
        <x>181</x>
        <y>185</y>
        <width>538</width>
        <height>380</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>538</width>
        <height>380</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>538</width>
        <height>380</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2.1px dashed #565656;
border-radius: 57px;</string>
      </property>
      <widget class="QPushButton" name="importPg_btn_spl">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>290</y>
         <width>251</width>
         <height>71</height>
        </rect>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">border: none;
background-color: transparent;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/icons/Splitter_GUI_Assets_3/Split Button Default.png</normaloff>:/icons/Splitter_GUI_Assets_3/Split Button Default.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>300</width>
         <height>101</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="importPg_btn_imp">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>290</y>
         <width>271</width>
         <height>71</height>
        </rect>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">border: none;
background-color: transparent;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/icons/Splitter_GUI_Assets_3/Browse Button Default.png</normaloff>:/icons/Splitter_GUI_Assets_3/Browse Button Default.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>300</width>
         <height>100</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="filesLogo">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>300</y>
        <width>150</width>
        <height>150</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0, 0, 0, 0); /* Set background color to transparent */
border-image: url(path/to/your/image.png) 10 10 10 10 stretch stretch; /* Adjust path to your image and border size */</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resource.qrc">:/icons/Splitter_GUI_Assets_3/File Drop Icon.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="splitPg">
     <widget class="Spin" name="splitPg_crl" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>900</width>
        <height>750</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="donePg">
     <widget class="QLabel" name="donePg_lbl2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>310</y>
        <width>311</width>
        <height>121</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Step 4: Process Completed</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="donePg_lbl">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>90</y>
        <width>391</width>
        <height>91</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Congrats or something</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="donePg_btn_newSplt">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>240</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Split new thing</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="licensePg">
     <widget class="QLabel" name="licensePg_lb">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>300</y>
        <width>251</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(86, 86, 86);</string>
      </property>
      <property name="text">
       <string>License Entry Page</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QTextEdit" name="licensePg_text">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>220</y>
        <width>211</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(86, 86, 86);</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Enter License Code Here&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="licensePg_cnf_btn">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>240</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(86, 86, 86);</string>
      </property>
      <property name="text">
       <string>Confirm</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="settingsPg">
     <widget class="QLabel" name="settingsPg_exp_lbl">
      <property name="geometry">
       <rect>
        <x>148</x>
        <y>468</y>
        <width>604</width>
        <height>55</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(86, 86, 86);
border: none;
border-radius: 15px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="settingsPg_stp_lbl">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>560</y>
        <width>211</width>
        <height>131</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(86, 86, 86);</string>
      </property>
      <property name="text">
       <string>Settings Page</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="settingsPg_licens_enter_btn">
      <property name="geometry">
       <rect>
        <x>148</x>
        <y>265</y>
        <width>604</width>
        <height>55</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(86, 86, 86);
border: none;
border-radius: 15px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="settingsPg_lic_lbl">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>230</y>
        <width>371</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(95, 95, 95);
font-family: &quot;Source Sans Pro&quot;;
font-size: 15pt; /* Adjust the font size as needed */
font-weight: bold; /* Set to bold */
font-style: italic; /* Set to italic */
background-color: transparent;</string>
      </property>
      <property name="text">
       <string>Please enter your license:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="settingsPg_exp_head_lbl">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>440</y>
        <width>451</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(95, 95, 95);
font-family: &quot;Source Sans Pro&quot;;
font-size: 15pt; /* Adjust the font size as needed */
font-weight: bold; /* Set to bold */
font-style: italic; /* Set to italic */
background-color: transparent;</string>
      </property>
      <property name="text">
       <string>Please enter a default export folder:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="settingsBtn">
    <property name="geometry">
     <rect>
      <x>827</x>
      <y>25</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/icons/Splitter_GUI_Assets_3/Settings Icon.png</normaloff>:/icons/Splitter_GUI_Assets_3/Settings Icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>77</width>
      <height>77</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="settingsPg_ext_btn">
    <property name="geometry">
     <rect>
      <x>827</x>
      <y>25</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background-color: transparent;</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/icons/Splitter_GUI_Assets_3/Exit Icon Default.png</normaloff>:/icons/Splitter_GUI_Assets_3/Exit Icon Default.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="settingsPg_exp_btn">
    <property name="geometry">
     <rect>
      <x>705</x>
      <y>476</y>
      <width>39</width>
      <height>39</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>39</width>
      <height>39</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>39</width>
      <height>39</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/icons/Splitter_GUI_Assets_3/Dots_Icon.png</normaloff>:/icons/Splitter_GUI_Assets_3/Dots_Icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>39</width>
      <height>39</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Drag</class>
   <extends>QWidget</extends>
   <header>dragFunction</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Spin</class>
   <extends>QWidget</extends>
   <header>spinnerFunction</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
